src/library_tty.js

```diff
@@ -161,6 +161,27 @@ mergeInto(LibraryManager.library, {
           out(UTF8ArrayToString(tty.output, 0));
           tty.output = [];
         }
+      },
+      ioctl_tcgets: function(tty) {
+        // typical setting
+        return {
+          c_iflag: 0x6500, // ICRNL | IXON | IMAXBEL | IUTF8
```

Comment:
Can you use `{{{ cDefs.ICRNL | cDefs.IXON | cDefs.IMAXBEL | cDefs.IUTF8 }}}` here instead of the hex number?  Or is that harder to read?

Reply:
Done at 1145e3975d190da10b1ee2c663f8da3ee8f535eb